tests/basic/storage_pool_test.go

```diff
@@ -2,6 +2,7 @@ package tests
 
 import (
 	"fmt"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Please keep the imports clean in sections

Reply:
again modified by goland